tests/conftest.py

```diff
@@ -77,4 +87,5 @@ def cleanup_db(db):
     yield
     schemas = db.get_schemas()
     for schema in schemas:
-        db.drop_schema(schema)
+        if schema not in [user.value for user in User]:
```

Comment:
In the previous commit there were the `guest` and `executor`  users. 

Not the `executor` and `admin` user are used.

Is that what you wanted?

Also, why not add the guest user as well in the enums?

Reply:
Great catch!